MonoZelda/Link/Projectiles/IProjectile.cs

```diff
@@ -6,6 +6,8 @@ public interface IProjectile
 {
 	void UpdateProjectile();
 
+	void FinishProjectile();
```

Comment:
It's possible there's a better name for this method than FinishProjectile, depending on what different projectiles' behavior is. Does finish mean that the projectile has hit something or reached a destination or otherwise needs to be destroyed? Destroy() could be better in that case.

(Also, this is a personal opinion, but I don't think Projectile method names should have projectile in them. For example, Projectile.Update is less redundant than Projectile.UpdateProjectile().)